docs/contributing.rst

```diff
@@ -450,3 +453,21 @@ Running Documentation Tests
 
 Make sure your contributions pass our documentation tests by running ``./scripts/docs.sh`` that installs dependencies
 needed for documentation and checks for any problems such as broken links or syntax issues.
+
+Solidity Language Design
+========================
+
+If you want to get involved in the language design process and share your ideas, please join the `solidity-users forum <https://groups.google.com/g/solidity-users>`_,
+where proposals for new language features and qualities of existing aspects of the language can be discussed.
```

Comment:
`qualities of existing aspects` sounds a bit odd to me

Reply:
True, we could just say `where aspects of the language and proposals for new language features can be discussed`?

Reply:
or better `where existing properties of the language and proposals for new....`?